frontend/src/components/history/attemptInfo.js

```diff
@@ -0,0 +1,145 @@
+import '../../App.css';
+import React from 'react';
+
+import Button from '@mui/material/Button';
+import Chip from '@mui/material/Chip';
+import { styled } from '@mui/material/styles';
+import Dialog from '@mui/material/Dialog';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import DialogActions from '@mui/material/DialogActions';
+import Typography from '@mui/material/Typography';
+
+import ComplexityChip from '../questions/complexityChip';
+
+const BootstrapDialog = styled(Dialog)(({ theme }) => ({
+  '& .MuiDialogContent-root': {
+    padding: theme.spacing(2),
+  },
+  '& .MuiDialogActions-root': {
+    padding: theme.spacing(1),
+  },
+}));
+
+function AttemptInfo({ open, handleClose, question, attempt }) {
+
+  // if there is no selected question, return null
+  if (question === undefined) {
+    return null;
+  }
+
+  const renderedParts = renderDescription(question.description);
+  const renderedAttempt = attempt;
+
+  return (
+    <div>
+      <BootstrapDialog
+        onClose={handleClose}
+        aria-labelledby="customized-dialog-title"
+        open={open}
+        fullWidth
+      >
+        {/* Question title */}
+        <DialogTitle sx={{ m: 0, p: 2 }} id="customized-dialog-title">
+          {question.title}
+        </DialogTitle>
+
+        {/* Rest of question contents in dialog box */}
+        <DialogContent dividers>
+          <Typography gutterBottom>
+            <b>ID:</b> {question.id}
+          </Typography>
+          <Typography gutterBottom sx={{ wordBreak: "break-word" }} component={'span'}>
+            <b>Category:</b> {question.categories.map((category) => (
+              <Chip key={category} label={category}></Chip>
+            ))}
+          </Typography>
+          <br />
+          <Typography gutterBottom sx={{ wordBreak: "break-word" }} component={'span'}>
+            <b>Complexity:</b> <ComplexityChip complexity={question.complexity} />
+          </Typography>
+          <Typography gutterBottom sx={{ whiteSpace: "pre-line" }} component="div">
+            {renderedParts}
```

Comment:
not sure if we need to display the entire description? or maybe swap this with attempt so users don't have to scroll all the way down to see their attempt, since attempt is the main focus

Reply:
i think it might be better if the description is displayed ont he left and the attempt on the right